src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfigs/cm_dotnet_minimal_api.json

```diff
@@ -75,16 +75,9 @@
               "Block": "WebApplication.CreateBuilder.Services.AddAuthorization()",
               "InsertAfter": "WebApplication.CreateBuilder.Services.AddAuthentication"
             },
-            {
-              "Block": "app.UseAuthentication()",
-              "InsertAfter": "app.UseHttpsRedirection",
-              "LeadingTrivia": {
-                "Newline": true
-              }
-            },
             {
               "Block": "app.UseAuthorization()",
-              "InsertAfter": "app.UseAuthentication"
```

Comment:
I think we should be able to remove `UseAuthorization` here as well since this particular configuration adds the authorization services here:

https://github.com/dotnet/Scaffolding/blob/057e3fd14c5dd433ed6070e543ee97533ac5e38e/src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfigs/cm_dotnet_minimal_api.json#L74-L77